website/src/parsers/php/php-parser.js

```diff
@@ -0,0 +1,40 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'php-parser/package.json';
+
+const ID = 'php-parser';
+
+const defaultOptions = {
+  parser: {
+    extractDoc: true,
+  },
+  ast: {
+    withPositions: true,
+  }
+};
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage,
+  locationProps: new Set(['loc']),
+
+  loadParser(callback) {
+    require(['php-parser'], callback);
+  },
+
+  parse(Engine, code, options) {
+    const parser = new Engine({...defaultOptions, ...options});
+    return parser.parseCode(code, '');
```

Comment:
Does error line highlighting work? If it doesn't currently, please see https://github.com/fkling/astexplorer/pull/247#issuecomment-310726688 for some tips on how to add that.